### PR TITLE
QubesOs blogpost

### PR DESCRIPTION
**Pull request by KBanecka**

### CODE REVIEW

File: blog/content/post/2020-05-14-qubesOs.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+  2. Michał Żygowski Anti Evil Maid for modern AMD UEFI-based platform
+* 04.06
+  1. Piotr Król DRTM for Qubes OS VMs
+  2. Michał Żygowski Anti Evil Maid for Intel coreboot-based platform
+* 10.06
+  1. Frédéric Pierret: How to build Qubes? From components to operating system
+overview.
+  2. Marek Marczykowski-Górecki Operating system testing, when it itself uses
+ virtual machines.
+
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-qubesOs.md:52]:
```suggestion
```
redundant new line

File: blog/content/post/2020-05-14-qubesOs.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+![QubesOs](/img/qubesos.png)
+
+### Whom will you meet?
+
+Prelections will be held by the team of 3mdeb experts who will share their
+knowledge and experience discussing AMD SEV and fwupd/LVFS, DRTMs and RTMs in
+Qubes OS. Among them are Piotr Król, founder and Embedded Systems Consultant at
+3mdeb as well as CTO of vitroTV, Michał Żygowski, 3mdeb core developer of
+Coreboot, maintainer of Braswell SoC, PC Engines, Protectli and Libretrend
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-qubesOs.md:74]:
```suggestion
coreboot, maintainer of Braswell SoC, PC Engines, Protectli and Libretrend
```
coreboot is always lower case.

File: blog/content/post/2020-05-14-qubesOs.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+platforms and Norbert Kamiński, Junior Embedded Systems Engineer focused on
+Linux embedded systems. Qubes OS testing and development will be widely
+presented by Qubes core developers, Marek Marczykowski-Górecki Project lead in
+QubesOS, specialized in everything Xen / Linux related and Frédéric Pierret,
+General packaging, CentOS and Fedora templates maintainer.  
+
+More about Qubes OS project you will find [here](https://www.qubes-os.org/).
+
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-qubesOs.md:82]:
```suggestion
```
redundant new line

File: blog/content/post/2020-05-14-qubesOs.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+### Why Qubes Os?
+
+To gain security of an Open Source quality.
```
Comment (miczyg1) [inline blog/content/post/2020-05-14-qubesOs.md:86]:
```suggestion
To gain security of an open-source quality.
```
I think we always write open-source this way

File: blog/content/post/2020-05-14-qubesOs.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
+---
+title: Qubes Os and 3mdeb 'minisummit' 2020
+abstract: 'Once again, we will meet on QubesOs & 3mdeb minisummit 2020 discussing #QubesOS,
+ #firmware, #coreboot, #security and #TPM related topics. All the event details are presented in the following blog post.'
```
[the remark this answers is not in the document]
Comment (artur-rs) [reply blog/content/post/2020-05-14-qubesOs.md:4]: @miczyg1 useful maybe not, but neither harmful in any way

File: blog/content/post/2020-05-14-qubesOs.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+cover: /covers/qubes&3mdeb_logo.png
+author: kamila.banecka
+layout: post
+published: true
+date: 2020-05-14
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:9]: @KBanecka please change the date in the header and filename to `2020-05-15`

File: blog/content/post/2020-05-14-qubesOs.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+  - Security
+  - Firmware
+
+
+---
+As we wrote [here](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/), recent Qubes OS 'minisummit' 2019 turned out to be fruitful and productive
+event. Great piece of knowledge has been presented and many ideas came up from
+long and deep discussions. Positively built with this experience, as observing
+changes inside security of firmare and computer architecture, we  couldn’t miss
+the opportunity and decided to organize Qubes OS minisummit, edition 2020. We
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:32]: @KBanecka please decide whether we are using `'minisummit'` or `minisummit`

File: blog/content/post/2020-05-14-qubesOs.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+want  to share our knowledge and our experience around topics presented below
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:33]: @KBanecka double space in `want  to`

File: blog/content/post/2020-05-14-qubesOs.md [hunk 9]
```diff
@@ -0,0 +1,129 @@
...
+  2. Norbert Kamiński Status fwupd/LVFS support for Qubes OS
+* 28.05
+  1. Piotr Król SRTM for Qubes OS VMs
+  2. Michał Żygowski Anti Evil Maid for modern AMD UEFI-based platform
+* 04.06
+  1. Piotr Król DRTM for Qubes OS VMs
+  2. Michał Żygowski Anti Evil Maid for Intel coreboot-based platform
+* 10.06
+  1. Frédéric Pierret: How to build Qubes? From components to operating system
+overview.
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:49]: @KBanecka missing indentation

File: blog/content/post/2020-05-14-qubesOs.md [hunk 10]
```diff
@@ -0,0 +1,129 @@
...
+
+Each stand will be divided into broad status report, design talk with proposals
+and discussion ended with  **Q&A** time for our viewers, so they could actively
+participate in the dialogue. We hope that the most promising presentations will
+be transformed into community supported or publicly founded projects.
+
+## Where to link the event?
+
+We are going to stream transmission via OBS on our YouTube 3mdeb channel. All
+you need to do is joining our [live](https://www.youtube.com/channel/UC_djHbyjuJvhVjfT18nyqmQ/live) in an appropriate time: all data details you will find below. Minisummit starts
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:62]: @KBanecka wrap text after the link - link can take more than 80 chars rule, but there shouldn't be any text in this line after the url

File: blog/content/post/2020-05-14-qubesOs.md [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+platforms and Norbert Kamiński, Junior Embedded Systems Engineer focused on
+Linux embedded systems. Qubes OS testing and development will be widely
+presented by Qubes core developers, Marek Marczykowski-Górecki Project lead in
+QubesOS, specialized in everything Xen / Linux related and Frédéric Pierret,
+General packaging, CentOS and Fedora templates maintainer.  
+
+More about Qubes OS project you will find [here](https://www.qubes-os.org/).
+
+
+### Why Qubes Os?
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:84]: @KBanecka OS?

File: blog/content/post/2020-05-14-qubesOs.md [hunk 12]
```diff
@@ -0,0 +1,129 @@
+---
+title: Qubes Os and 3mdeb 'minisummit' 2020
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:2]: @KBanecka OS

File: blog/content/post/2020-05-14-qubesOs.md [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+
+>I'm excited about upcoming Qubes OS minisummit, happening thanks to 3mdeb
+folks. There will be many interesting topics presented by competent people. And
+also a lot of potential for further collaboration!
+
+> -- <cite>Marek Marczykowski-Górecki</cite>
+
+## Summary
+
+We are all waiting for Qubes Os minisummit 2020 where the 3mdeb experts will cover
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md]: @KBanecka OS

File: blog/content/post/2020-05-14-qubesOs.md [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+topics related with status fwupd/LVFS support, SRTM and DTRM for Qubes OS and
+Anti Evil Maid for Intel coreboot-based platform. Qubes OS specialists will
+tell us how to build and test operating system that uses virtual
+machines. Every discussion will end with Q&A session where participants will have the opportunity to join discussion. At this point, awaiting for the event, we want to thank both the experts who will devote their
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md]: @KBanecka wrap to 80

File: blog/content/post/2020-05-14-qubesOs.md [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+time to sharing knowledge and experience, but also the participants of the
+event, passionate about QubesOs, without which the event would not make sense.
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md]: @KBanecka why QubesOs without space? Is it intended?

File: blog/content/post/2020-05-14-qubesOs.md [hunk 16]
```diff
@@ -0,0 +1,138 @@
...
+
+>I'm excited about upcoming Qubes OS minisummit, happening thanks to 3mdeb
+folks. There will be many interesting topics presented by competent people. And
+also a lot of potential for further collaboration!
+
+> -- <cite>Marek Marczykowski-Górecki</cite>
+
+## Summary
+
+We are all waiting for Qubes Os minisummit 2020 where the 3mdeb experts will
```
Comment (pietrushnic) [inline blog/content/post/2020-05-14-qubesOs.md:125]:
```suggestion
We are all waiting for Qubes OS minisummit 2020 where the 3mdeb experts will
```

File: blog/content/post/2020-05-14-qubesOs.md [hunk 17]
```diff
@@ -0,0 +1,138 @@
...
+  - security
+  - TPM
+categories:
+  - Miscellaneous
+  - Security
+  - Firmware
+
+
+---
+Recent Qubes OS 'minisummit'
```
Comment (marmarek) [inline blog/content/post/2020-05-14-qubesOs.md:28]:
```suggestion
Previous Qubes OS 'minisummit'
```
Almost a year ago I wouldn't call "recent"

File: blog/content/post/2020-05-14-qubesOs.md (hunk 1 above)
Comment (KBanecka) [reply blog/content/post/2020-05-14-qubesOs.md:53]: shouldn't be a new line between list and paragraph?

File: blog/content/post/2020-05-14-qubesOs.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (KBanecka) [reply blog/content/post/2020-05-14-qubesOs.md:4]: Shall I leave it?

[the remark this answers is not in the document]
Comment (artur-rs) [reply blog/content/post/2020-05-14-qubesOs.md:4]: @KBanecka it's fine, let's keep them

File: blog/content/post/2020-05-14-qubesOs.md [hunk 18]
```diff
@@ -0,0 +1,141 @@
...
+Previous Qubes OS 'minisummit'
+[2019](https://blog.3mdeb.com/2019/2019-08-07-qubes-os-and-3mdeb-minisummit/)
+turned out to be fruitful and productive event. Great piece of knowledge has
+been presented and many ideas came up from long and deep discussions. Positively
+built with this experience, as observing changes inside security of firmware and
+computer architecture, we couldn’t miss the opportunity and decided to organize
+Qubes OS 'minisummit', edition 2020. We want to share our knowledge and our
+experience around topics presented below hoping, that once more, we go far
+beyond the agenda. This year, we will stand side by side with Frédéric Pierret
+and Marek Marczykowski Górecki from [QubesOS](https://www.qubes-os.org/) team, covering the following topics:
```
Comment (artur-rs) [inline blog/content/post/2020-05-14-qubesOs.md:37]: @KBanecka wrap 80

File: blog/content/post/2020-05-14-qubesOs.md (hunk 6 above)
Comment (artur-rs) [reply blog/content/post/2020-05-14-qubesOs.md]: @KBanecka still filename has old date

File: blog/content/post/2020-05-14-qubesOs.md (hunk 15 above)
Comment (artur-rs) [reply blog/content/post/2020-05-14-qubesOs.md]: @KBanecka ^

File: blog/content/post/2020-05-14-qubesOs.md [hunk 19]
```diff
@@ -0,0 +1,138 @@
...
+* 10.06
+  1. Frédéric Pierret: How to build Qubes? From components to operating system
+     overview.
+  2. Marek Marczykowski-Górecki Operating system testing, when it itself uses
+     virtual machines.
+
+Each stand will be divided into broad status report, design talk with proposals
+and discussion ended with  **Q&A** time for our viewers, so they could actively
+participate in the dialogue. We hope that the most promising presentations will
+be transformed into community supported or publicly founded projects.
```
[the remark this answers is not in the document]
Comment (artur-rs) [reply blog/content/post/2020-05-14-qubesOs.md]: funded - financial support, founded - launched. @KBanecka if you are not making the requested changes, please response to the comment with your decision/argumentation

**Pull request merged**